backend/src/packages/users/user.repository.ts

```diff
@@ -26,6 +26,19 @@ class UserRepository implements IRepository {
     return Promise.resolve([]);
   }
 
+  public async findByPhone(value: string): Promise<UserEntity | undefined> {
```

Comment:
it's better to return UserEntity | null

Reply:
'null' I've changed, but Promise can't remove because type problem arrears